src/funding/applepay/config.jsx

```diff
@@ -0,0 +1,59 @@
+/* @flow */
+/** @jsx node */
+
+import { PLATFORM } from '@paypal/sdk-constants/src';
+import { getUserAgent, isIOS, isSafari } from 'belter';
+import { ApplePayLogo, LOGO_COLOR } from '@paypal/sdk-logos/src';
+
+import { BUTTON_COLOR, BUTTON_LAYOUT } from '../../constants';
+import { DEFAULT_FUNDING_CONFIG, type FundingSourceConfig } from '../common';
+
+export function getApplePayConfig() : FundingSourceConfig {
+    return {
+        ...DEFAULT_FUNDING_CONFIG,
+
+        requiresPopupSupport:             false,
+        requiresSupportedNativeBrowser:   false,
+        shippingChange:                   true,
+
+        platforms: [
+            PLATFORM.DESKTOP,
+            PLATFORM.MOBILE
+        ],
+
+        layouts: [
```

Comment:
Are we supporting both layouts for Apple Pay?

Reply:
No harm in it.